app/auth-portal/src/content/tutorial/08-asset_authoring.md

```diff
@@ -0,0 +1,442 @@
+---
+title: Asset Authoring
+---
+
+## How to create an Asset in System Initiative
+
+Today, we are going to build a System Initiative asset that allows us to create and manage
+[AWS EBS Volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volumes.html). We are going to create a 100GB EBS Volume in AWS Ohio (us-east-2a)
+that has 3000 iops, running on an SSD and is unencrypted.
+
+**Please take care to follow the guide precisely; there's a lot of rough edges. Specifically,
+ensure you stick with the function naming scheme and setting the appropriate properties of
+attached functions.**
+
+### Anatomy of an Asset
+
+An Asset is a collection of functions that come together to provide the capability to talk
+to an infrastructure provider, e.g. an AWS EBS Volume. An asset is made of the following
+types of functions:
+
+* Schema Variant Definition
+* Qualification
+* Action
+* Code Generation
+* Attribute
+* Validation
+
```

Comment:
I'd put these in the order that you will address them below, so it's easier to follow along.

Reply:
Maybe also good to write a short sentence on what each of them are? Will help people understand what an asset is.

Reply:
I am staying away from the descriptions of the types for this post - TL;DR, that will be a different post